docs/development/intro.md

```diff
@@ -1,6 +1,6 @@
 # Development
 
-Developers who wants to debug the application have to open 2 terminals.
+Developers who wants to debug the application have to open two terminals.
 
 In first terminal run `yarn run dev` to start webpack-dev for front-end developing and hot reloading at <http://localhost:8092>
```

Comment:
Nit, but `run` isn't necessary when executing package scripts with `yarn`. You can just use `yarn dev` etc.